### ISSUE TITLE
Error when writing `:w` file when tidy.nvim is enabled

### ISSUE THREAD

**Issue by sirutBuasai** [mcauley-penney/tidy.nvim#14]
Received an error when writing file.
```
Error detected while processing BufWritePre Autocommands for "*":
Error executing lua callback: ...config/nvim/autoload/plugged/tidy.nvim/lua/tidy/init.lua:17: attempt to call field 'list_contains' (a nil value)
stack traceback:
        ...config/nvim/autoload/plugged/tidy.nvim/lua/tidy/init.lua:17: in function 'is_excluded_ft'
```

**Comment by mcauley-penney** [comment]
Thanks for the issue! I made my commit this morning using the [Lua documentation](https://neovim.io/doc/user/lua.html) but didn't realize that [vim.list_contains() is only on nightly](https://github.com/neovim/neovim/commit/4d04feb6629cb049cb2a13ba35f0c8d3c6b67ff4). That's my bad. I'll revert it and move it to branch for nvim 0.10

**Comment by mcauley-penney** [comment]
Should be closed by 86eab24e807ed2957d26ca43b2b76b92d0fbf8f4. Let me know if you have any issues
